src/site/content/en/react/quicklink-react-spa/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: Speed up navigations in React with quicklink
```

Comment:
```suggestion
title: Speed up navigations in React with Quicklink
```
Let's format Quicklink with uppercase throughout the article. In the Google Doc draft I said to format it like `quicklink` but I'm realizing now that that idea is kind of silly.